unittest/formats/test_atom_styles.cpp

```diff
@@ -2743,6 +2743,7 @@ TEST_F(AtomStyleTest, template)
     expected.has_x         = true;
     expected.has_v         = true;
     expected.has_f         = true;
+    expected.q_flag        = 1;
```

Comment:
This change must be reverted. See my comment in the corresponding AtomVec class

Reply:
done